pom.xml

```diff
@@ -602,6 +621,11 @@
                     <artifactId>maven-surefire-plugin</artifactId>
                     <version>${maven-surefire-plugin.version}</version>
                 </plugin>
+                <plugin>
```

Comment:
both the maven surefire and failsafe plugins need to be inherited from the parent pom and should not need to be listed here.

Reply:
Correct! I was just finishing the last tests before making the commits. I'm going to use the `codeQuality` profile to call the maven surefire and failsafe plugins.